jest/mockModal.js

```diff
@@ -13,11 +13,10 @@
 'use strict';
 
 const React = require('react');
-const Modal = require('../Libraries/Modal/Modal');
```

Comment:
Would changing this to `import typeof` instead also work without introducing `$FlowFixMe`?

Reply:
You're right, importing the type works fine

Reply:
You're right, it works fine when importing the type directly